src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -60,23 +61,27 @@
 public class ExperimentProcessor implements Processor {
 
     private static final String NAME = "Experiment";
+    private static final String missingObsUnitIdError = "Experiment Units are missing Observation Unit Id.\n" +
```

Comment:
Minor quibble:  By convention, because it is a static final,  the variable should be in all upper case (MISSING_OBS_UNIT_ID_ERROR)

Reply:
Good catch! Fixed.